accord-core/src/main/java/accord/messages/Commit.java

```diff
@@ -162,7 +162,8 @@ public synchronized ReadNack apply(SafeCommandStore safeStore)
             case Insufficient:
                 SafeCommand safeCommand = safeStore.command(txnId);
                 Invariants.checkState(!safeCommand.current().known().isDefinitionKnown());
-                if (defer == null)
+                // When isDone=true this.process() is called and all CommandStores run apply again; so safe to create a new one
```

Comment:
this is from https://issues.apache.org/jira/browse/CASSANDRA-18364, I hit in burn test so patched here to get stable